### ISSUE TITLE
Install fails due to django python requirement that cannot be installed

### ISSUE THREAD

**Issue by madchap** [DefectDojo/django-DefectDojo#956]
Please use the following template when submitting issues: (If this is a security issue, then review the [disclosure policy](https://github.com/DefectDojo/django-DefectDojo/blob/master/SECURITY.md) and submit via our security reporting program.

### Issue Type 
- [x] Bug
- [ ] Enhancement

### Description
Could not find a version that satisfies the requirement Django==1.11.19 (from DefectDojo===1.5.3)

The installation fails.

### Operating System 
Please include the version number (ex: Ubuntu 16.04)
Ubuntu 16.04 and 18.04

### Install

- [x] setup.bash
- [ ] docker

### DefectDojo Version
git master

### Steps to Reproduce
Just run setup.bash

### Console Logs (optional)
```
Collecting celery==4.1.1 (from DefectDojo===1.5.3)
  Downloading https://files.pythonhosted.org/packages/99/fa/4049b26bfe71992ecf979acd39b87e55b493608613054089d975418015b7/celery-4.1.1-py2.py3-none-any.whl (394kB)
    100% |████████████████████████████████| 399kB 2.8MB/s
Collecting defusedxml (from DefectDojo===1.5.3)
  Downloading https://files.pythonhosted.org/packages/87/1c/17f3e3935a913dfe2a5ca85fa5ccbef366bfd82eb318b1f75dadbf0affca/defusedxml-0.5.0-py2.py3-none-any.whl
Collecting Django==1.11.19 (from DefectDojo===1.5.3)
  Could not find a version that satisfies the requirement Django==1.11.19 (from DefectDojo===1.5.3) (from versions: 1.1.3, 1.1.4, 1.2, 1.2.1, 1.2.2, 1.2.3, 1.2.4, 1.2.5, 1.2.6, 1.2.7, 1.3, 1.3.1, 1.3.2, 1.3.3, 1.3.4, 1.3.5, 1.3.6, 1.3.7, 1.4, 1.4.1, 1.4.2, 1.4.3, 1.4.4, 1.4.5, 1.4.6, 1.4.7, 1.4.8, 1.4.9, 1.4.10, 1.4.11, 1.4.12, 1.4.13, 1.4.14, 1.4.15, 1.4.16, 1.4.17, 1.4.18, 1.4.19, 1.4.20, 1.4.21, 1.4.22, 1.5, 1.5.1, 1.5.2, 1.5.3, 1.5.4, 1.5.5, 1.5.6, 1.5.7, 1.5.8, 1.5.9, 1.5.10, 1.5.11, 1.5.12, 1.6, 1.6.1, 1.6.2, 1.6.3, 1.6.4, 1.6.5, 1.6.6, 1.6.7, 1.6.8, 1.6.9, 1.6.10, 1.6.11, 1.7, 1.7.1, 1.7.2, 1.7.3, 1.7.4, 1.7.5, 1.7.6, 1.7.7, 1.7.8, 1.7.9, 1.7.10, 1.7.11, 1.8a1, 1.8b1, 1.8b2, 1.8rc1, 1.8, 1.8.1, 1.8.2, 1.8.3, 1.8.4, 1.8.5, 1.8.6, 1.8.7, 1.8.8, 1.8.9, 1.8.10, 1.8.11, 1.8.12, 1.8.13, 1.8.14, 1.8.15, 1.8.16, 1.8.17, 1.8.18, 1.8.19, 1.9a1, 1.9b1, 1.9rc1, 1.9rc2, 1.9, 1.9.1, 1.9.2, 1.9.3, 1.9.4, 1.9.5, 1.9.6, 1.9.7, 1.9.8, 1.9.9, 1.9.10, 1.9.11, 1.9.12, 1.9.13, 1.10a1, 1.10b1, 1.10rc1, 1.10, 1.10.1, 1.10.2, 1.10.3, 1.10.4, 1.10.5, 1.10.6, 1.10.7, 1.10.8, 1.11a1, 1.11b1, 1.11rc1, 1.11, 1.11.1, 1.11.2, 1.11.3, 1.11.4, 1.11.5, 1.11.6, 1.11.7, 1.11.8, 1.11.9, 1.11.10, 1.11.11, 1.11.12, 1.11.13, 1.11.14, 1.11.15, 1.11.16, 1.11.17, 1.11.18, 1.11.20, 2.0, 2.0.1, 2.0.2, 2.0.3, 2.0.4, 2.0.5, 2.0.6, 2.0.7, 2.0.8, 2.0.9, 2.0.10, 2.0.12, 2.0.13, 2.1, 2.1.1, 2.1.2, 2.1.3, 2.1.4, 2.1.5, 2.1.7, 2.2a1, 2.2b1, 2.2rc1)
No matching distribution found for Django==1.11.19 (from DefectDojo===1.5.3)
```

**Comment by aaronweaver** [comment]
@madchap That's fixed now with 1.11.20

**Comment by madchap** [comment]
for follow-up's sake:

* Checked out stable tag `git checkout tags/1.5.3.1 -b 1.5.3.1`
* Change requirements.txt Django version. Diff is:

```
diff --git a/requirements.txt b/requirements.txt
index 002f77d..b290f53 100644
--- a/requirements.txt
+++ b/requirements.txt
@@ -1,7 +1,7 @@
 celery==4.1.0
 coverage==4.5.1
 defusedxml==0.5.0
-Django==1.11.11
+Django==1.11.20
 django-auditlog==0.4.5
 django-custom-field==2.9
 django-filter==1.0.4
```

That went through OK it seems.
